nginx/conf.d/emgapi.conf

```diff
@@ -1,48 +0,0 @@
-ignore_invalid_headers off;
```

Comment:
Has this file been replaced or did it become obsolete.

Reply:
that is local dev stuff, I'll add JIRA task to clean stuff.